cost_map:
  global_frame: /map
  robot_base_frame: /base_link
  update_frequency: 2.0
  publish_frequency: 2.0 
  static_map: true
  rolling_window: false
  track_unknown_space: true
  always_send_full_costmap: false
  transform_tolerance: 0.2
  resolution: 0.05 
 

  plugins: 
    - {name: static_layer,         type: "costmap_2d::StaticLayer"}
    - {name: inflation_layer,      type: "costmap_2d::InflationLayer"}
  

  static_layer: 
    map_topic: /map
    unknown_cost_value: 255  #default=-1
    trinary_costmap: true
    lethal_cost_threshold: 100 #default=100
    track_unknown_space: true
    use_maximum: false
    first_map_only: true #default=false
    subscribe_to_updates: false  #if true, it subscribes to topic /map_topic_updates
 

  inflation_layer:
    inflation_radius: 0.35 #1.0
    cost_scaling_factor: 10.0

   
  footprint: [ [0.345, 0], [0.244, -0.244], [0, -0.345], [-0.244, -0.244], [-0.345, 0], [-0.244, 0.244], [0, 0.345], [0.244, 0.244]]
  transform_tolerance: 0.5 
  map_type: costmap


